### PR TITLE
Investigate compatibility with libmamba 1.5.0

### PR DESCRIPTION
**Pull request by jaimergp**
### Description

One test broke with libmamba 1.5.0: `test_solvers.py::test_defaults_specs_work`. libmamba is not able to handle `defaults::*` as a matchspec, but in <=1.4.9 we could workaround by translating defaults to the most likely `pkgs/*` equivalent. This workaround doesn't work in 1.5.0 anymore so I am simply xfailing the test and adding a cleaner exception to the UI.

This is reported upstream at https://github.com/mamba-org/mamba/issues/2431, but no solution is available yet.

### Checklist - did you ...

- [x] Add a file to the `news` directory ([using the template](../blob/main/news/TEMPLATE)) for the next release's release notes?
     
- [X] Add / update necessary tests?
- [ ] Add / update outdated documentation?

### CODE REVIEW

**Comment by jaimergp** [comment]
Errors are unrelated (see #267 and #242). This doesn't fix the underlying issue but does annotate the offending test as xfail. Work is needed upstream for a proper fix.

File: conda_libmamba_solver/solver.py [hunk 1]
```diff
@@ -364,7 +364,21 @@ def _solve_attempt(
...
+                if mamba_version() == "1.5.0":
+                    for spec in specs:
+                        if spec in str(exc):
+                            break
+                    else:
+                        spec = f"One of {specs}"
+                    msg = (
+                        "This is a bug in libmamba 1.5.0. 'defaults' (and 'pkgs/*') cannot be "
+                        "used in channel::pkg_name specs. Please use -c defaults instead."
+                    )
```
Comment (jezdez) [inline conda_libmamba_solver/solver.py:379]: While technically correct, is this really something we want to bubble up to users?

```suggestion
                    msg = (
                        "This is a bug in libmamba 1.5.0 when using 'defaults::"
                        "<spec>' or 'pkgs/main::<spec>'. Please use '-c "
                        " defaults' instead."
                    )
```

Is `pkgs/main::<spec>` correct?

**Review by jezdez — APPROVED** [review]
Just minor nit

File: conda_libmamba_solver/solver.py (hunk 1 above)
Comment (jaimergp) [reply conda_libmamba_solver/solver.py:379]:
```suggestion
                    msg = (
                        "This is a bug in libmamba 1.5.0 when using 'defaults::"
                        "<spec>' or 'pkgs/main::<spec>'. Please use '-c "
                        "defaults' instead."
                    )
```

**Pull request merged**